src/style.css

```diff
@@ -0,0 +1,92 @@
+/* 20220620 properties "background" structured for better understanding */
+/* 20220620 selector "input" changed to class "textSearch" */
+/* 20220620 added color to class "textSearch" and attribute "placeholder" */
+/* 20220622 deleted over data attributes */
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: bold;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+}
+
+.searchBar {
+  margin: 20px 10px;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61,78,97,0.1);
+
+  height: 70px;
+}
+
+.searchBar[data-qa="small"] {
+  height: 42px;
+}
+
+.textSearch {
+  height: 100%;
+  border: none;
+  color: #3d4e61;
+
+  padding: 0 0 0 33px;
+  width: calc(100% - 33px);
+  font-size: 14px;
+  background: url(images/Search.svg)
+    no-repeat
+    left
+    15px/40px 11px;
+}
+
+.textSearch[data-qa="keypress"] {
```

Comment:
you're still using data attributes. I've wrote you a comment last time to get rid of them, please remove them and use classes only